packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -309,26 +309,17 @@ export const makeEditGenerator = (
 		const { content: field } = fieldInfo;
 		assert(field.length > 0, "Sequence must have at least one element to perform a move");
 
-		// This can be done in O(1) but it's more clear this way:
-		// Valid move indices are any index before or equal to the start of the sequence
-		// and after the end of the sequence.
 		const start = state.random.integer(0, field.length - 1);
 		const count = state.random.integer(1, field.length - start);
-		const validMoveIndices: number[] = [];
-		for (let i = 0; i < field.length; i++) {
-			if (i <= start || i > start + count) {
-				validMoveIndices.push(i);
-			}
-		}
-		const moveIndex = state.random.pick(validMoveIndices);
+		const dstIndex = state.random.integer(0, field.length);
```

Comment:
What is the outcome if we have a sequence with 3 nodes and we move all 3 of the nodes to index 1/2? Do we get "filler" or do they just end up staying where they were?

Reply:
They stay where they are when you don't consider empty cells, which is to say they move to a new set of cells and the cells of origin are empty (and therefore invisible to the host application). This kind of operation is unlikely but it is valid and we need to support it.